.github/workflows/staging-resources-cdk-check.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow checks the Staging Resources CDK project
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Staging Resources CDK App Check
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'release/staging-resources-cdk/**'
+  pull_request:
+    paths:
+      - 'release/staging-resources-cdk/**'
+  workflow_dispatch:
```

Comment:
What does `workflow_dispatch` do without `inputs`?
Like here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions